ministryofjustice/correspondence_tool_staff
# CT-2847 Make the "request info" as optional

## Description
Make the "request info" box as optional when creating offender_sar case 

### Related JIRA tickets
https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=CT&modal=detail&selectedIssue=CT-2847

## Self-review checklist

 
### Screenshots

Before: 
![Screenshot 2020-05-13 at 14 23 31](https://user-images.githubusercontent.com/26111671/81818493-009ff400-9526-11ea-9c63-bd0a3ea7548c.png)
After 
![Screenshot 2020-05-13 at 11 21 53](https://user-images.githubusercontent.com/26111671/81818554-144b5a80-9526-11ea-85fc-67ed1f09bcfb.png)
![Screenshot 2020-05-13 at 17 22 44](https://user-images.githubusercontent.com/26111671/81839347-50d78000-953f-11ea-8f2a-13b5d3aa1917.png)

Click change
![Screenshot 2020-05-13 at 11 21 22](https://user-images.githubusercontent.com/26111671/81818590-20cfb300-9526-11ea-8e98-a511e679286d.png)
![Screenshot 2020-05-13 at 11 21 33](https://user-images.githubusercontent.com/26111671/81818603-262cfd80-9526-11ea-82cd-de73c09eede1.png)

### Deployment

 No need

### Manual testing instructions

1 - login as Brian.rix
2 - Create a offender_sar case
3 - when reaching to the page of ```/cases/offender_sars/requested-info```
      leave it as blank and click "Continue" until to the end
4 - Once the case detail page is shown and move down to the request box, should see the message of "Use this space to store additional case information." in the box
